### DISCUSSION TITLE
Beginner question: how to customize the header to remove top nav bar?

### DISCUSSION THREAD

**Discussion by tuokaikyle** [transitive-bullshit/nextjs-notion-starter-kit#420] [Q&A]
Hey thanks for creating this repo, it's really cool.

I duplicated the default page as a starting point, and was wondering how can I remove the nav bar at the very top (the part that above the background image, and contains breadcrumbs and searchbox)? 

I can achieve it from the css side, but just wondering how to do it on the jsx part, hope to get some help, thanks!

**Comment by eddiehe99** [comment]
You can now customize the header on the latest version.
